FirstSolution/Part9Task3/Part9Task3.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Part9Task3
+{
+    
+    // At a given text string, define the number of words. Each word is separated
+    // from each other by a space.
+    
+    public class Task
+    {
+        private static void Main(string[] args)
+        {
+            string input = Console.ReadLine();
+            string text = input.Trim();
+            string[] words = text.Split(' ');
```

Comment:
Try to write this commands into one line

Reply:
Done